src/app/components/main-welcome/main-welcome.component.styl

```diff
@@ -14,15 +14,16 @@
     #user-guide
       width: 550px
       text-align: right
-    
+  .list-column
```

Comment:
could we make the `welcome-projects-list` wider (width:100%) and then maybe make these columns take up that space (but give less space to the 3rd column).  maybe a follow on ticket?